src/main/java/codeu/controller/RegisterServletTest.java

```diff
@@ -0,0 +1,46 @@
+/*Anjuli*/
+package codeu.controller;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+public class RegisterServletTest {
+    private RegisterServlet registerServlet;
+    private HttpServletRequest mockRequest;
+    private PrintWriter mockPrintWriter;
+    private HttpServletResponse mockResponse;
+    private RequestDispatcher mockRequestDispatcher;
+
+    @Before
+    public void setup() throws IOException{
+        registerServlet = new RegisterServlet();
+
+        mockRequest = Mockito.mock(HttpServletRequest.class);
+        mockResponse = Mockito.mock(HttpServletResponse.class);
+        mockRequestDispatcher= Mockito.mock(RequestDispatcher.class);
+
+        Mockito.when(mockRequest.getRequestDispatcher("/WEB-INF/view/register.jsp"))
+                .thenReturn(mockRequestDispatcher);
+
+        //mockPrintWriter = Mockito.mock(PrintWriter.class);
```

Comment:
remove these commented-out lines (and previous comments about using @Mock apply)